go.test.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+echo "" > coverage.txt
+
+for d in $(go list ./... | grep -v vendor); do
+    go test -race -coverprofile=profile.out -covermode=atomic $d
```

Comment:
[Data Race Detector \- The Go Programming Language](https://golang.org/doc/articles/race_detector.html)
But we don't need it. I'll remove `-race` option. Thanks.